presto-hive/src/test/java/com/facebook/presto/hive/TestHiveMaterializedViewLogicalPlanner.java

```diff
@@ -1882,6 +1882,82 @@ public void testMaterializedViewOptimizationWithDoublePartition()
         }
     }
 
+    @Test
```

Comment:
This test would work previously as well, right?

Reply:
Yes